### PR TITLE
Sliced A100 GPUs

### PR DESCRIPTION
**Pull request by galfthan**
## Proposed changes

Mahti will have A100 that have been sliced into smaller GPUs.

https://csc-guide-preview.rahtiapp.fi/origin/A100_1G.5GB/

## Checklist before requesting a review

- [x] I have followed the instructions in the [Contributing](https://github.com/CSCfi/csc-user-guide/blob/master/CONTRIBUTING.md) and [Styleguide](https://github.com/CSCfi/csc-user-guide/blob/master/STYLEGUIDE.md) documents.
- [x] My pull request passes all tests.
- [x] I have included a link to the appropriate [preview page](https://csc-guide-preview.rahtiapp.fi/origin/) (select your branch from the list).

### CODE REVIEW

**Review by CSC-swesters — CHANGES_REQUESTED** [review]
Made a note about CPU core limits + a few GRES name typos, but otherwise good! If it was not intended to be the GRES name itself in fluent text, maybe we should make the "non-GRES" name for the slices more distinct, so that users don't use the wrong name in Slurm job scripts?

File: docs/computing/webinterface/index.md [hunk 1]
```diff
@@ -174,6 +174,18 @@ started and you will be able to connect to the application on the _My Interactiv
...
+#### Partitions and resources
+
+Only some of the partitions on the system are available for use in the web interfaces. Some apps
+have a more limited set of partitions available than other apps.
+
+In the Puhti web interface, the `interactive`, `small`, `test`, `gpu` and `gputest` partitions are available.
+Selecting the `gpu` or `gputest` partition will allocate a Nvidia V100 GPU.
+
+In the Mahti web interface, the `interactive` and `gpusmall` partitions are available. Selecting the
+`gpusmall` partition will allocate a split Nvidia A100 GPU (A100_1g.5g). For more details about the
```
Comment (CSC-swesters) [inline docs/computing/webinterface/index.md:186]: > A100_1g.5g

Change to `a100_1g.5gb` (notice "gb" as well).

File: docs/computing/running/batch-job-partitions.md [hunk 2]
```diff
@@ -69,7 +69,9 @@ The following partition is available for allocation by resources in **Mahti** on
 |-----------|-------------|---------------|------------------|------------------|
 | [interactive](interactive-usage.md#sinteractive-in-mahti)  |  1-32      |  7 days      | 60 GiB           | all              |
 
-The following partitions are available on GPU nodes. Note that for each GPU, you can reserve at most 32 cores/task and for each GPU the job is allocated 122.5 GiB of memory.
+The following partitions are available on GPU nodes. Note that for each full A100 GPU, you can reserve at most 32 cores/task and for each GPU the job is allocated 122.5 GiB of memory. 
+A subset of the A100 GPUs are sliced into smaller A100_1g.5gb GPUs with one seventh of the compute and memory capacity of a full A100 GPU. For each small GPU you can reserve at most 
+4 cores/task and for each GPU the job is allocated 17.5 GiB of memory. Also note that you can reserve at most one small GPU per job.
```
[the remark this answers is not in the document]
Comment (galfthan) [reply docs/computing/running/batch-job-partitions.md:74]: I meant to write cores/job actually, so the cores per job is at most 4. 
 Will fix that way

File: docs/computing/webinterface/index.md (hunk 1 above)
Comment (galfthan) [reply docs/computing/webinterface/index.md]: fixed

File: docs/computing/running/creating-job-scripts-mahti.md [hunk 3]
```diff
@@ -77,7 +77,19 @@ Mahti's `gpusmall` partition supports only one or two GPU jobs. So the maximum i
 #SBATCH --partition=gpusmall
 #SBATCH --gres=gpu:a100:1
 ```
-Mahti's `gpumedium` partition will support multi-GPU jobs and only four GPUs per compute node. 
+
+In Mahti's `gpusmall` partition there are also A100 GPUs that have been sliced into smaller A100_1g.5g GPUs 
+with one seventh of the compute and memory capacity of a full A100 GPU.  For each small GPU you can reserve 
+at most 4 cores/task and for each GPU the job is allocated 17.5 GiB of memory. Also note that you can reserve
```
[the remark this answers is not in the document]
Comment (CSC-swesters) [reply docs/computing/running/creating-job-scripts-mahti.md:83]: This line still has a `cores/task`, while it's probably meant to be `cores/tasks`?

**Review by rkronberg — APPROVED** [review]

**Pull request merged**